[Java][Native] remove the warnings

##### Description

Found the following warning in the Java native client:
```
[WARNING] Javadoc Warnings
[WARNING] /private/var/tmp/openapi-generator/samples/client/petstore/java/native-async/src/main/java/org/openapitools/client/JSON.java:63: warning: no @return
[WARNING] public static Class<?> getClassForElement(JsonNode node, Class<?> modelClass) {
[WARNING] ^
[WARNING] /private/var/tmp/openapi-generator/samples/client/petstore/java/native-async/src/main/java/org/openapitools/client/JSON.java:168: warning: no @param for visitedClasses
[WARNING] public static boolean isInstanceOf(Class<?> modelClass, Object inst, Set<Class<?>> visitedClasses) {
[WARNING] ^
[WARNING] /private/var/tmp/openapi-generator/samples/client/petstore/java/native-async/src/main/java/org/openapitools/client/JSON.java:168: warning: no @return
[WARNING] public static boolean isInstanceOf(Class<?> modelClass, Object inst, Set<Class<?>> visitedClasses) {
[WARNING] ^
[WARNING] /private/var/tmp/openapi-generator/samples/client/petstore/java/native-async/src/main/java/org/openapitools/client/api/FakeApi.java:84: warning: no @return
[WARNING] public CompletableFuture<Void> createXmlItem(XmlItem xmlItem) throws ApiException {
[WARNING] ^
[WARNING] /private/var/tmp/openapi-generator/samples/client/petstore/java/native-async/src/main/java/org/openapitools/client/api/FakeApi.java:519: warning: no @return
[WARNING] public CompletableFuture<Void> testBodyWithFileSchema(FileSchemaTestClass body) throws ApiException {
[WARNING] ^
[WARNING] /private/var/tmp/openapi-generator/samples/client/petstore/java/native-async/src/main/java/org/openapitools/client/api/FakeApi.java:599: warning: no @return
[WARNING] public CompletableFuture<Void> testBodyWithQueryParams(String query, User body) throws ApiException {
[WARNING] ^
[WARNING] /private/var/tmp/openapi-generator/samples/client/petstore/java/native-async/src/main/java/org/openapitools/client/api/FakeApi.java:798: warning: no @return
[WARNING] public CompletableFuture<Void> testEndpointParameters(BigDecimal number, Double _double, String patternWithoutDelimiter, byte[] _byte, Integer integer, Integer int32, Long int64, Float _float, String string, File binary, LocalDate date, OffsetDateTime dateTime, String password, String paramCallback) throws ApiException {
[WARNING] ^
[WARNING] /private/var/tmp/openapi-generator/samples/client/petstore/java/native-async/src/main/java/org/openapitools/client/api/FakeApi.java:903: warning: no @return
[WARNING] public CompletableFuture<Void> testEnumParameters(List<String> enumHeaderStringArray, String enumHeaderString, List<String> enumQueryStringArray, String enumQueryString, Integer enumQueryInteger, Double enumQueryDouble, List<String> enumFormStringArray, String enumFormString) throws ApiException {
[WARNING] ^
[WARNING] /private/var/tmp/openapi-generator/samples/client/petstore/java/native-async/src/main/java/org/openapitools/client/api/FakeApi.java:997: warning: no @return
[WARNING] public CompletableFuture<Void> testGroupParameters(APItestGroupParametersRequest apiRequest) throws ApiException {
[WARNING] ^
[WARNING] /private/var/tmp/openapi-generator/samples/client/petstore/java/native-async/src/main/java/org/openapitools/client/api/FakeApi.java:1035: warning: no @return
[WARNING] public CompletableFuture<Void> testGroupParameters(Integer requiredStringGroup, Boolean requiredBooleanGroup, Long requiredInt64Group, Integer stringGroup, Boolean booleanGroup, Long int64Group) throws ApiException {
[WARNING] ^
[WARNING] /private/var/tmp/openapi-generator/samples/client/petstore/java/native-async/src/main/java/org/openapitools/client/api/FakeApi.java:1216: warning: no @return
[WARNING] public CompletableFuture<Void> testInlineAdditionalProperties(Map<String, String> param) throws ApiException {
[WARNING] ^
[WARNING] /private/var/tmp/openapi-generator/samples/client/petstore/java/native-async/src/main/java/org/openapitools/client/api/FakeApi.java:1296: warning: no @return
[WARNING] public CompletableFuture<Void> testJsonFormData(String param, String param2) throws ApiException {
[WARNING] ^
[WARNING] /private/var/tmp/openapi-generator/samples/client/petstore/java/native-async/src/main/java/org/openapitools/client/api/FakeApi.java:1378: warning: no @return
[WARNING] public CompletableFuture<Void> testQueryParameterCollectionFormat(List<String> pipe, List<String> ioutil, List<String> http, List<String> url, List<String> context) throws ApiException {
[WARNING] ^
[WARNING] /private/var/tmp/openapi-generator/samples/client/petstore/java/native-async/src/main/java/org/openapitools/client/api/PetApi.java:79: warning: no @return
[WARNING] public CompletableFuture<Void> addPet(Pet body) throws ApiException {
[WARNING] ^
[WARNING] /private/var/tmp/openapi-generator/samples/client/petstore/java/native-async/src/main/java/org/openapitools/client/api/PetApi.java:159: warning: no @return
[WARNING] public CompletableFuture<Void> deletePet(Long petId, String apiKey) throws ApiException {
[WARNING] ^
[WARNING] /private/var/tmp/openapi-generator/samples/client/petstore/java/native-async/src/main/java/org/openapitools/client/api/PetApi.java:521: warning: no @return
[WARNING] public CompletableFuture<Void> updatePet(Pet body) throws ApiException {
[WARNING] ^
[WARNING] /private/var/tmp/openapi-generator/samples/client/petstore/java/native-async/src/main/java/org/openapitools/client/api/PetApi.java:602: warning: no @return
[WARNING] public CompletableFuture<Void> updatePetWithForm(Long petId, String name, String status) throws ApiException {
[WARNING] ^
[WARNING] /private/var/tmp/openapi-generator/samples/client/petstore/java/native-async/src/main/java/org/openapitools/client/api/StoreApi.java:76: warning: no @return
[WARNING] public CompletableFuture<Void> deleteOrder(String orderId) throws ApiException {
[WARNING] ^
[WARNING] /private/var/tmp/openapi-generator/samples/client/petstore/java/native-async/src/main/java/org/openapitools/client/api/UserApi.java:76: warning: no @return
[WARNING] public CompletableFuture<Void> createUser(User body) throws ApiException {
[WARNING] ^
[WARNING] /private/var/tmp/openapi-generator/samples/client/petstore/java/native-async/src/main/java/org/openapitools/client/api/UserApi.java:155: warning: no @return
[WARNING] public CompletableFuture<Void> createUsersWithArrayInput(List<User> body) throws ApiException {
[WARNING] ^
[WARNING] /private/var/tmp/openapi-generator/samples/client/petstore/java/native-async/src/main/java/org/openapitools/client/api/UserApi.java:234: warning: no @return
[WARNING] public CompletableFuture<Void> createUsersWithListInput(List<User> body) throws ApiException {
[WARNING] ^
[WARNING] /private/var/tmp/openapi-generator/samples/client/petstore/java/native-async/src/main/java/org/openapitools/client/api/UserApi.java:313: warning: no @return
[WARNING] public CompletableFuture<Void> deleteUser(String username) throws ApiException {
[WARNING] ^
[WARNING] /private/var/tmp/openapi-generator/samples/client/petstore/java/native-async/src/main/java/org/openapitools/client/api/UserApi.java:577: warning: no @return
[WARNING] public CompletableFuture<Void> logoutUser() throws ApiException {
[WARNING] ^
[WARNING] /private/var/tmp/openapi-generator/samples/client/petstore/java/native-async/src/main/java/org/openapitools/client/api/UserApi.java:646: warning: no @return
[WARNING] public CompletableFuture<Void> updateUser(String username, User body) throws ApiException {
[WARNING] ^
[WARNING] /private/var/tmp/openapi-generator/samples/client/petstore/java/native-async/src/main/java/org/openapitools/client/model/AdditionalPropertiesAnyType.java:83: warning: no @param for key
[WARNING] public AdditionalPropertiesAnyType putAdditionalProperty(String key, Object value) {
[WARNING] ^
[WARNING] /private/var/tmp/openapi-generator/samples/client/petstore/java/native-async/src/main/java/org/openapitools/client/model/AdditionalPropertiesAnyType.java:83: warning: no @param for value
[WARNING] public AdditionalPropertiesAnyType putAdditionalProperty(String key, Object value) {
[WARNING] ^
[WARNING] /private/var/tmp/openapi-generator/samples/client/petstore/java/native-async/src/main/java/org/openapitools/client/model/AdditionalPropertiesAnyType.java:83: warning: no @return
[WARNING] public AdditionalPropertiesAnyType putAdditionalProperty(String key, Object value) {
[WARNING] ^
[WARNING] /private/var/tmp/openapi-generator/samples/client/petstore/java/native-async/src/main/java/org/openapitools/client/model/AdditionalPropertiesAnyType.java:95: warning: no @return
[WARNING] public Map<String, Object> getAdditionalProperties() {
[WARNING] ^
[WARNING] /private/var/tmp/openapi-generator/samples/client/petstore/java/native-async/src/main/java/org/openapitools/client/model/AdditionalPropertiesAnyType.java:102: warning: no @param for key
[WARNING] public Object getAdditionalProperty(String key) {
[WARNING] ^
[WARNING] /private/var/tmp/openapi-generator/samples/client/petstore/java/native-async/src/main/java/org/openapitools/client/model/AdditionalPropertiesAnyType.java:102: warning: no @return
[WARNING] public Object getAdditionalProperty(String key) {
[WARNING] ^
[WARNING] /private/var/tmp/openapi-generator/samples/client/petstore/java/native-async/src/main/java/org/openapitools/client/model/AdditionalPropertiesArray.java:84: warning: no @param for key
[WARNING] public AdditionalPropertiesArray putAdditionalProperty(String key, List value) {
[WARNING] ^
[WARNING] /private/var/tmp/openapi-generator/samples/client/petstore/java/native-async/src/main/java/org/openapitools/client/model/AdditionalPropertiesArray.java:84: warning: no @param for value
[WARNING] public AdditionalPropertiesArray putAdditionalProperty(String key, List value) {
[WARNING] ^
[WARNING] /private/var/tmp/openapi-generator/samples/client/petstore/java/native-async/src/main/java/org/openapitools/client/model/AdditionalPropertiesArray.java:84: warning: no @return
[WARNING] public AdditionalPropertiesArray putAdditionalProperty(String key, List value) {
[WARNING] ^
[WARNING] /private/var/tmp/openapi-generator/samples/client/petstore/java/native-async/src/main/java/org/openapitools/client/model/AdditionalPropertiesArray.java:96: warning: no @return
[WARNING] public Map<String, List> getAdditionalProperties() {
[WARNING] ^
[WARNING] /private/var/tmp/openapi-generator/samples/client/petstore/java/native-async/src/main/java/org/openapitools/client/model/AdditionalPropertiesArray.java:103: warning: no @param for key
[WARNING] public List getAdditionalProperty(String key) {
[WARNING] ^
[WARNING] /private/var/tmp/openapi-generator/samples/client/petstore/java/native-async/src/main/java/org/openapitools/client/model/AdditionalPropertiesArray.java:103: warning: no @return
[WARNING] public List getAdditionalProperty(String key) {
[WARNING] ^
[WARNING] /private/var/tmp/openapi-generator/samples/client/petstore/java/native-async/src/main/java/org/openapitools/client/model/AdditionalPropertiesBoolean.java:83: warning: no @param for key
[WARNING] public AdditionalPropertiesBoolean putAdditionalProperty(String key, Boolean value) {
[WARNING] ^
[WARNING] /private/var/tmp/openapi-generator/samples/client/petstore/java/native-async/src/main/java/org/openapitools/client/model/AdditionalPropertiesBoolean.java:83: warning: no @param for value
[WARNING] public AdditionalPropertiesBoolean putAdditionalProperty(String key, Boolean value) {
[WARNING] ^
[WARNING] /private/var/tmp/openapi-generator/samples/client/petstore/java/native-async/src/main/java/org/openapitools/client/model/AdditionalPropertiesBoolean.java:83: warning: no @return
[WARNING] public AdditionalPropertiesBoolean putAdditionalProperty(String key, Boolean value) {
[WARNING] ^
[WARNING] /private/var/tmp/openapi-generator/samples/client/petstore/java/native-async/src/main/java/org/openapitools/client/model/AdditionalPropertiesBoolean.java:95: warning: no @return
[WARNING] public Map<String, Boolean> getAdditionalProperties() {
[WARNING] ^
[WARNING] /private/var/tmp/openapi-generator/samples/client/petstore/java/native-async/src/main/java/org/openapitools/client/model/AdditionalPropertiesBoolean.java:102: warning: no @param for key
[WARNING] public Boolean getAdditionalProperty(String key) {
[WARNING] ^
[WARNING] /private/var/tmp/openapi-generator/samples/client/petstore/java/native-async/src/main/java/org/openapitools/client/model/AdditionalPropertiesBoolean.java:102: warning: no @return
[WARNING] public Boolean getAdditionalProperty(String key) {
[WARNING] ^
[WARNING] /private/var/tmp/openapi-generator/samples/client/petstore/java/native-async/src/main/java/org/openapitools/client/model/AdditionalPropertiesInteger.java:83: warning: no @param for key
[WARNING] public AdditionalPropertiesInteger putAdditionalProperty(String key, Integer value) {
[WARNING] ^
[WARNING] /private/var/tmp/openapi-generator/samples/client/petstore/java/native-async/src/main/java/org/openapitools/client/model/AdditionalPropertiesInteger.java:83: warning: no @param for value
[WARNING] public AdditionalPropertiesInteger putAdditionalProperty(String key, Integer value) {
[WARNING] ^
[WARNING] /private/var/tmp/openapi-generator/samples/client/petstore/java/native-async/src/main/java/org/openapitools/client/model/AdditionalPropertiesInteger.java:83: warning: no @return
[WARNING] public AdditionalPropertiesInteger putAdditionalProperty(String key, Integer value) {
[WARNING] ^
[WARNING] /private/var/tmp/openapi-generator/samples/client/petstore/java/native-async/src/main/java/org/openapitools/client/model/AdditionalPropertiesInteger.java:95: warning: no @return
[WARNING] public Map<String, Integer> getAdditionalProperties() {
[WARNING] ^
[WARNING] /private/var/tmp/openapi-generator/samples/client/petstore/java/native-async/src/main/java/org/openapitools/client/model/AdditionalPropertiesInteger.java:102: warning: no @param for key
[WARNING] public Integer getAdditionalProperty(String key) {
[WARNING] ^
[WARNING] /private/var/tmp/openapi-generator/samples/client/petstore/java/native-async/src/main/java/org/openapitools/client/model/AdditionalPropertiesInteger.java:102: warning: no @return
[WARNING] public Integer getAdditionalProperty(String key) {
[WARNING] ^
[WARNING] /private/var/tmp/openapi-generator/samples/client/petstore/java/native-async/src/main/java/org/openapitools/client/model/AdditionalPropertiesNumber.java:84: warning: no @param for key
[WARNING] public AdditionalPropertiesNumber putAdditionalProperty(String key, BigDecimal value) {
[WARNING] ^
[WARNING] /private/var/tmp/openapi-generator/samples/client/petstore/java/native-async/src/main/java/org/openapitools/client/model/AdditionalPropertiesNumber.java:84: warning: no @param for value
[WARNING] public AdditionalPropertiesNumber putAdditionalProperty(String key, BigDecimal value) {
[WARNING] ^
[WARNING] /private/var/tmp/openapi-generator/samples/client/petstore/java/native-async/src/main/java/org/openapitools/client/model/AdditionalPropertiesNumber.java:84: warning: no @return
[WARNING] public AdditionalPropertiesNumber putAdditionalProperty(String key, BigDecimal value) {
[WARNING] ^
[WARNING] /private/var/tmp/openapi-generator/samples/client/petstore/java/native-async/src/main/java/org/openapitools/client/model/AdditionalPropertiesNumber.java:96: warning: no @return
[WARNING] public Map<String, BigDecimal> getAdditionalProperties() {
[WARNING] ^
[WARNING] /private/var/tmp/openapi-generator/samples/client/petstore/java/native-async/src/main/java/org/openapitools/client/model/AdditionalPropertiesNumber.java:103: warning: no @param for key
[WARNING] public BigDecimal getAdditionalProperty(String key) {
[WARNING] ^
[WARNING] /private/var/tmp/openapi-generator/samples/client/petstore/java/native-async/src/main/java/org/openapitools/client/model/AdditionalPropertiesNumber.java:103: warning: no @return
[WARNING] public BigDecimal getAdditionalProperty(String key) {
[WARNING] ^
[WARNING] /private/var/tmp/openapi-generator/samples/client/petstore/java/native-async/src/main/java/org/openapitools/client/model/AdditionalPropertiesObject.java:83: warning: no @param for key
[WARNING] public AdditionalPropertiesObject putAdditionalProperty(String key, Map value) {
[WARNING] ^
[WARNING] /private/var/tmp/openapi-generator/samples/client/petstore/java/native-async/src/main/java/org/openapitools/client/model/AdditionalPropertiesObject.java:83: warning: no @param for value
[WARNING] public AdditionalPropertiesObject putAdditionalProperty(String key, Map value) {
[WARNING] ^
[WARNING] /private/var/tmp/openapi-generator/samples/client/petstore/java/native-async/src/main/java/org/openapitools/client/model/AdditionalPropertiesObject.java:83: warning: no @return
[WARNING] public AdditionalPropertiesObject putAdditionalProperty(String key, Map value) {
[WARNING] ^
[WARNING] /private/var/tmp/openapi-generator/samples/client/petstore/java/native-async/src/main/java/org/openapitools/client/model/AdditionalPropertiesObject.java:95: warning: no @return
[WARNING] public Map<String, Map> getAdditionalProperties() {
[WARNING] ^
[WARNING] /private/var/tmp/openapi-generator/samples/client/petstore/java/native-async/src/main/java/org/openapitools/client/model/AdditionalPropertiesObject.java:102: warning: no @param for key
[WARNING] public Map getAdditionalProperty(String key) {
[WARNING] ^
[WARNING] /private/var/tmp/openapi-generator/samples/client/petstore/java/native-async/src/main/java/org/openapitools/client/model/AdditionalPropertiesObject.java:102: warning: no @return
[WARNING] public Map getAdditionalProperty(String key) {
[WARNING] ^
[WARNING] /private/var/tmp/openapi-generator/samples/client/petstore/java/native-async/src/main/java/org/openapitools/client/model/AdditionalPropertiesString.java:83: warning: no @param for key
[WARNING] public AdditionalPropertiesString putAdditionalProperty(String key, String value) {
[WARNING] ^
[WARNING] /private/var/tmp/openapi-generator/samples/client/petstore/java/native-async/src/main/java/org/openapitools/client/model/AdditionalPropertiesString.java:83: warning: no @param for value
[WARNING] public AdditionalPropertiesString putAdditionalProperty(String key, String value) {
[WARNING] ^
[WARNING] /private/var/tmp/openapi-generator/samples/client/petstore/java/native-async/src/main/java/org/openapitools/client/model/AdditionalPropertiesString.java:83: warning: no @return
[WARNING] public AdditionalPropertiesString putAdditionalProperty(String key, String value) {
[WARNING] ^
[WARNING] /private/var/tmp/openapi-generator/samples/client/petstore/java/native-async/src/main/java/org/openapitools/client/model/AdditionalPropertiesString.java:95: warning: no @return
[WARNING] public Map<String, String> getAdditionalProperties() {
[WARNING] ^
[WARNING] /private/var/tmp/openapi-generator/samples/client/petstore/java/native-async/src/main/java/org/openapitools/client/model/AdditionalPropertiesString.java:102: warning: no @param for key
[WARNING] public String getAdditionalProperty(String key) {
[WARNING] ^
[WARNING] /private/var/tmp/openapi-generator/samples/client/petstore/java/native-async/src/main/java/org/openapitools/client/model/AdditionalPropertiesString.java:102: warning: no @return
[WARNING] public String getAdditionalProperty(String key) {
[WARNING] ^
```


##### openapi-generator version

Latest master

##### OpenAPI declaration file content or url

Petstore

##### Generation Details

<!-- 
    Prefer CLI steps, including the language, libraries and various options. 
    Providing config-based settings allows for simpler testing across CLI and plugins. 
    For examples, see https://github.com/OpenAPITools/openapi-generator/tree/master/bin/configs
-->

##### Steps to reproduce

```
mvn integration-test -f samples/client/petstore/java/native-async/pom.xml
```

##### Related issues/PRs

https://github.com/OpenAPITools/openapi-generator/pull/8554

##### Suggest a fix

Will need to update the mustache templates to fix these issues

--- a/modules/openapi-generator/src/main/resources/Java/libraries/native/additional_properties.mustache
+++ b/modules/openapi-generator/src/main/resources/Java/libraries/native/additional_properties.mustache
@@ -9,6 +9,9 @@
   /**
    * Set the additional (undeclared) property with the specified name and value.
    * If the property does not already exist, create it otherwise replace it.
+   * @param key the name of the property
+   * @param value the value value of the property
+   * @return self reference
    */
   @JsonAnySetter
   public {{classname}} putAdditionalProperty(String key, {{{.}}} value) {
@@ -20,7 +23,8 @@
   }
 
   /**
-   * Return the additional (undeclared) property.
+   * Return the additional (undeclared) properties.
+   * @return the additional (undeclared) properties
    */
   @JsonAnyGetter
   public Map<String, {{{.}}}> getAdditionalProperties() {
@@ -29,6 +33,8 @@
 
   /**
    * Return the additional (undeclared) property with the specified name.
+   * @param key the name of the property
+   * @return the additional (undeclared) property with the specified name
    */
   public {{{.}}} getAdditionalProperty(String key) {
     if (this.additionalProperties == null) {

--- a/modules/openapi-generator/src/main/resources/Java/libraries/native/api.mustache
+++ b/modules/openapi-generator/src/main/resources/Java/libraries/native/api.mustache
@@ -91,6 +91,11 @@ public class {{classname}} {
    {{#returnType}}
    * @return {{#asyncNative}}CompletableFuture&lt;{{/asyncNative}}{{returnType}}{{#asyncNative}}&gt;{{/asyncNative}}
    {{/returnType}}
+   {{^returnType}}
+   {{#asyncNative}}
+   * @return CompletableFuture&lt;Void&gt;
+   {{/asyncNative}}
+   {{/returnType}}
    * @throws ApiException if fails to make API call
    {{#isDeprecated}}
    * @deprecated
@@ -144,6 +149,11 @@ public class {{classname}} {
    {{/allParams}}
    {{#returnType}}
    * @return {{#asyncNative}}CompletableFuture&lt;{{/asyncNative}}{{returnType}}{{#asyncNative}}&gt;{{/asyncNative}}
+   {{/returnType}}
+   {{^returnType}}
+   {{#asyncNative}}
+   * @return CompletableFuture&lt;Void&gt;
+   {{/asyncNative}}
    {{/returnType}}
    * @throws ApiException if fails to make API call
    {{#isDeprecated}}

--- a/samples/client/petstore/java/native-async/src/main/java/org/openapitools/client/api/FakeApi.java
+++ b/samples/client/petstore/java/native-async/src/main/java/org/openapitools/client/api/FakeApi.java
@@ -87,6 +87,7 @@ public class FakeApi {
    * creates an XmlItem
    * this route creates an XmlItem
    * @param xmlItem XmlItem Body (required)
+   * @return CompletableFuture&lt;Void&gt;
    * @throws ApiException if fails to make API call
    */
   public CompletableFuture<Void> createXmlItem(XmlItem xmlItem) throws ApiException {
@@ -532,6 +533,7 @@ public class FakeApi {
    * 
    * For this test, the body for this request much reference a schema named &#x60;File&#x60;.
    * @param body  (required)
+   * @return CompletableFuture&lt;Void&gt;
    * @throws ApiException if fails to make API call
    */
   public CompletableFuture<Void> testBodyWithFileSchema(FileSchemaTestClass body) throws ApiException {
@@ -615,6 +617,7 @@ public class FakeApi {
    * 
    * @param query  (required)
    * @param body  (required)
+   * @return CompletableFuture&lt;Void&gt;
    * @throws ApiException if fails to make API call
    */
   public CompletableFuture<Void> testBodyWithQueryParams(String query, User body) throws ApiException {
@@ -820,6 +823,7 @@ public class FakeApi {
    * @param dateTime None (optional)
    * @param password None (optional)
    * @param paramCallback None (optional)
+   * @return CompletableFuture&lt;Void&gt;
    * @throws ApiException if fails to make API call
    */
   public CompletableFuture<Void> testEndpointParameters(BigDecimal number, Double _double, String patternWithoutDelimiter, byte[] _byte, Integer integer, Integer int32, Long int64, Float _float, String string, File binary, LocalDate date, OffsetDateTime dateTime, String password, String paramCallback) throws ApiException {
@@ -928,6 +932,7 @@ public class FakeApi {
    * @param enumQueryDouble Query parameter enum test (double) (optional)
    * @param enumFormStringArray Form parameter enum test (string array) (optional
    * @param enumFormString Form parameter enum test (string) (optional, default to -efg)
+   * @return CompletableFuture&lt;Void&gt;
    * @throws ApiException if fails to make API call
    */
   public CompletableFuture<Void> testEnumParameters(List<String> enumHeaderStringArray, String enumHeaderString, List<String> enumQueryStringArray, String enumQueryString, Integer enumQueryInteger, Double enumQueryDouble, List<String> enumFormStringArray, String enumFormString) throws ApiException {
@@ -1025,6 +1030,7 @@ public class FakeApi {
    * Fake endpoint to test group parameters (optional)
    * Fake endpoint to test group parameters (optional)
    * @param apiRequest {@link APItestGroupParametersRequest}
+   * @return CompletableFuture&lt;Void&gt;
    * @throws ApiException if fails to make API call
    */
   public CompletableFuture<Void> testGroupParameters(APItestGroupParametersRequest apiRequest) throws ApiException {
@@ -1063,6 +1069,7 @@ public class FakeApi {
    * @param stringGroup String in group parameters (optional)
    * @param booleanGroup Boolean in group parameters (optional)
    * @param int64Group Integer in group parameters (optional)
+   * @return CompletableFuture&lt;Void&gt;
    * @throws ApiException if fails to make API call
    */
   public CompletableFuture<Void> testGroupParameters(Integer requiredStringGroup, Boolean requiredBooleanGroup, Long requiredInt64Group, Integer stringGroup, Boolean booleanGroup, Long int64Group) throws ApiException {
@@ -1247,6 +1254,7 @@ public class FakeApi {
    * test inline additionalProperties
    * 
    * @param param request body (required)
+   * @return CompletableFuture&lt;Void&gt;
    * @throws ApiException if fails to make API call
    */
   public CompletableFuture<Void> testInlineAdditionalProperties(Map<String, String> param) throws ApiException {
@@ -1330,6 +1338,7 @@ public class FakeApi {
    * 
    * @param param field1 (required)
    * @param param2 field2 (required)
+   * @return CompletableFuture&lt;Void&gt;
    * @throws ApiException if fails to make API call
    */
   public CompletableFuture<Void> testJsonFormData(String param, String param2) throws ApiException {
@@ -1415,6 +1424,7 @@ public class FakeApi {
    * @param http  (required)
    * @param url  (required)
    * @param context  (required)
+   * @return CompletableFuture&lt;Void&gt;
    * @throws ApiException if fails to make API call
    */
   public CompletableFuture<Void> testQueryParameterCollectionFormat(List<String> pipe, List<String> ioutil, List<String> http, List<String> url, List<String> context) throws ApiException {

--- a/samples/client/petstore/java/native-async/src/main/java/org/openapitools/client/api/PetApi.java
+++ b/samples/client/petstore/java/native-async/src/main/java/org/openapitools/client/api/PetApi.java
@@ -82,6 +82,7 @@ public class PetApi {
    * Add a new pet to the store
    * 
    * @param body Pet object that needs to be added to the store (required)
+   * @return CompletableFuture&lt;Void&gt;
    * @throws ApiException if fails to make API call
    */
   public CompletableFuture<Void> addPet(Pet body) throws ApiException {
@@ -165,6 +166,7 @@ public class PetApi {
    * 
    * @param petId Pet id to delete (required)
    * @param apiKey  (optional)
+   * @return CompletableFuture&lt;Void&gt;
    * @throws ApiException if fails to make API call
    */
   public CompletableFuture<Void> deletePet(Long petId, String apiKey) throws ApiException {
@@ -539,6 +541,7 @@ public class PetApi {
    * Update an existing pet
    * 
    * @param body Pet object that needs to be added to the store (required)
+   * @return CompletableFuture&lt;Void&gt;
    * @throws ApiException if fails to make API call
    */
   public CompletableFuture<Void> updatePet(Pet body) throws ApiException {
@@ -623,6 +626,7 @@ public class PetApi {
    * @param petId ID of pet that needs to be updated (required)
    * @param name Updated name of the pet (optional)
    * @param status Updated status of the pet (optional)
+   * @return CompletableFuture&lt;Void&gt;
    * @throws ApiException if fails to make API call
    */
   public CompletableFuture<Void> updatePetWithForm(Long petId, String name, String status) throws ApiException {

--- a/samples/client/petstore/java/native-async/src/main/java/org/openapitools/client/api/StoreApi.java
+++ b/samples/client/petstore/java/native-async/src/main/java/org/openapitools/client/api/StoreApi.java
@@ -79,6 +79,7 @@ public class StoreApi {
    * Delete purchase order by ID
    * For valid response try integer IDs with value &lt; 1000. Anything above 1000 or nonintegers will generate API errors
    * @param orderId ID of the order that needs to be deleted (required)
+   * @return CompletableFuture&lt;Void&gt;
    * @throws ApiException if fails to make API call
    */
   public CompletableFuture<Void> deleteOrder(String orderId) throws ApiException {

--- a/samples/client/petstore/java/native-async/src/main/java/org/openapitools/client/api/UserApi.java
+++ b/samples/client/petstore/java/native-async/src/main/java/org/openapitools/client/api/UserApi.java
@@ -80,6 +80,7 @@ public class UserApi {
    * Create user
    * This can only be done by the logged in user.
    * @param body Created user object (required)
+   * @return CompletableFuture&lt;Void&gt;
    * @throws ApiException if fails to make API call
    */
   public CompletableFuture<Void> createUser(User body) throws ApiException {
@@ -162,6 +163,7 @@ public class UserApi {
    * Creates list of users with given input array
    * 
    * @param body List of user object (required)
+   * @return CompletableFuture&lt;Void&gt;
    * @throws ApiException if fails to make API call
    */
   public CompletableFuture<Void> createUsersWithArrayInput(List<User> body) throws ApiException {
@@ -244,6 +246,7 @@ public class UserApi {
    * Creates list of users with given input array
    * 
    * @param body List of user object (required)
+   * @return CompletableFuture&lt;Void&gt;
    * @throws ApiException if fails to make API call
    */
   public CompletableFuture<Void> createUsersWithListInput(List<User> body) throws ApiException {
@@ -326,6 +329,7 @@ public class UserApi {
    * Delete user
    * This can only be done by the logged in user.
    * @param username The name that needs to be deleted (required)
+   * @return CompletableFuture&lt;Void&gt;
    * @throws ApiException if fails to make API call
    */
   public CompletableFuture<Void> deleteUser(String username) throws ApiException {
@@ -599,6 +603,7 @@ public class UserApi {
   /**
    * Logs out current logged in user session
    * 
+   * @return CompletableFuture&lt;Void&gt;
    * @throws ApiException if fails to make API call
    */
   public CompletableFuture<Void> logoutUser() throws ApiException {
@@ -671,6 +676,7 @@ public class UserApi {
    * This can only be done by the logged in user.
    * @param username name that need to be deleted (required)
    * @param body Updated user object (required)
+   * @return CompletableFuture&lt;Void&gt;
    * @throws ApiException if fails to make API call
    */
   public CompletableFuture<Void> updateUser(String username, User body) throws ApiException {

--- a/samples/client/petstore/java/native-async/src/main/java/org/openapitools/client/model/AdditionalPropertiesAnyType.java
+++ b/samples/client/petstore/java/native-async/src/main/java/org/openapitools/client/model/AdditionalPropertiesAnyType.java
@@ -82,6 +82,9 @@ public class AdditionalPropertiesAnyType extends HashMap<String, Object> {
   /**
    * Set the additional (undeclared) property with the specified name and value.
    * If the property does not already exist, create it otherwise replace it.
+   * @param key the name of the property
+   * @param value the value value of the property
+   * @return self reference
    */
   @JsonAnySetter
   public AdditionalPropertiesAnyType putAdditionalProperty(String key, Object value) {
@@ -93,7 +96,8 @@ public class AdditionalPropertiesAnyType extends HashMap<String, Object> {
   }
 
   /**
-   * Return the additional (undeclared) property.
+   * Return the additional (undeclared) properties.
+   * @return the additional (undeclared) properties
    */
   @JsonAnyGetter
   public Map<String, Object> getAdditionalProperties() {
@@ -102,6 +106,8 @@ public class AdditionalPropertiesAnyType extends HashMap<String, Object> {
 
   /**
    * Return the additional (undeclared) property with the specified name.
+   * @param key the name of the property
+   * @return the additional (undeclared) property with the specified name
    */
   public Object getAdditionalProperty(String key) {
     if (this.additionalProperties == null) {

--- a/samples/client/petstore/java/native-async/src/main/java/org/openapitools/client/model/AdditionalPropertiesArray.java
+++ b/samples/client/petstore/java/native-async/src/main/java/org/openapitools/client/model/AdditionalPropertiesArray.java
@@ -83,6 +83,9 @@ public class AdditionalPropertiesArray extends HashMap<String, List> {
   /**
    * Set the additional (undeclared) property with the specified name and value.
    * If the property does not already exist, create it otherwise replace it.
+   * @param key the name of the property
+   * @param value the value value of the property
+   * @return self reference
    */
   @JsonAnySetter
   public AdditionalPropertiesArray putAdditionalProperty(String key, List value) {
@@ -94,7 +97,8 @@ public class AdditionalPropertiesArray extends HashMap<String, List> {
   }
 
   /**
-   * Return the additional (undeclared) property.
+   * Return the additional (undeclared) properties.
+   * @return the additional (undeclared) properties
    */
   @JsonAnyGetter
   public Map<String, List> getAdditionalProperties() {
@@ -103,6 +107,8 @@ public class AdditionalPropertiesArray extends HashMap<String, List> {
 
   /**
    * Return the additional (undeclared) property with the specified name.
+   * @param key the name of the property
+   * @return the additional (undeclared) property with the specified name
    */
   public List getAdditionalProperty(String key) {
     if (this.additionalProperties == null) {

--- a/samples/client/petstore/java/native-async/src/main/java/org/openapitools/client/model/AdditionalPropertiesBoolean.java
+++ b/samples/client/petstore/java/native-async/src/main/java/org/openapitools/client/model/AdditionalPropertiesBoolean.java
@@ -82,6 +82,9 @@ public class AdditionalPropertiesBoolean extends HashMap<String, Boolean> {
   /**
    * Set the additional (undeclared) property with the specified name and value.
    * If the property does not already exist, create it otherwise replace it.
+   * @param key the name of the property
+   * @param value the value value of the property
+   * @return self reference
    */
   @JsonAnySetter
   public AdditionalPropertiesBoolean putAdditionalProperty(String key, Boolean value) {
@@ -93,7 +96,8 @@ public class AdditionalPropertiesBoolean extends HashMap<String, Boolean> {
   }
 
   /**
-   * Return the additional (undeclared) property.
+   * Return the additional (undeclared) properties.
+   * @return the additional (undeclared) properties
    */
   @JsonAnyGetter
   public Map<String, Boolean> getAdditionalProperties() {
@@ -102,6 +106,8 @@ public class AdditionalPropertiesBoolean extends HashMap<String, Boolean> {
 
   /**
    * Return the additional (undeclared) property with the specified name.
+   * @param key the name of the property
+   * @return the additional (undeclared) property with the specified name
    */
   public Boolean getAdditionalProperty(String key) {
     if (this.additionalProperties == null) {

--- a/samples/client/petstore/java/native-async/src/main/java/org/openapitools/client/model/AdditionalPropertiesInteger.java
+++ b/samples/client/petstore/java/native-async/src/main/java/org/openapitools/client/model/AdditionalPropertiesInteger.java
@@ -82,6 +82,9 @@ public class AdditionalPropertiesInteger extends HashMap<String, Integer> {
   /**
    * Set the additional (undeclared) property with the specified name and value.
    * If the property does not already exist, create it otherwise replace it.
+   * @param key the name of the property
+   * @param value the value value of the property
+   * @return self reference
    */
   @JsonAnySetter
   public AdditionalPropertiesInteger putAdditionalProperty(String key, Integer value) {
@@ -93,7 +96,8 @@ public class AdditionalPropertiesInteger extends HashMap<String, Integer> {
   }
 
   /**
-   * Return the additional (undeclared) property.
+   * Return the additional (undeclared) properties.
+   * @return the additional (undeclared) properties
    */
   @JsonAnyGetter
   public Map<String, Integer> getAdditionalProperties() {
@@ -102,6 +106,8 @@ public class AdditionalPropertiesInteger extends HashMap<String, Integer> {
 
   /**
    * Return the additional (undeclared) property with the specified name.
+   * @param key the name of the property
+   * @return the additional (undeclared) property with the specified name
    */
   public Integer getAdditionalProperty(String key) {
     if (this.additionalProperties == null) {

--- a/samples/client/petstore/java/native-async/src/main/java/org/openapitools/client/model/AdditionalPropertiesNumber.java
+++ b/samples/client/petstore/java/native-async/src/main/java/org/openapitools/client/model/AdditionalPropertiesNumber.java
@@ -83,6 +83,9 @@ public class AdditionalPropertiesNumber extends HashMap<String, BigDecimal> {
   /**
    * Set the additional (undeclared) property with the specified name and value.
    * If the property does not already exist, create it otherwise replace it.
+   * @param key the name of the property
+   * @param value the value value of the property
+   * @return self reference
    */
   @JsonAnySetter
   public AdditionalPropertiesNumber putAdditionalProperty(String key, BigDecimal value) {
@@ -94,7 +97,8 @@ public class AdditionalPropertiesNumber extends HashMap<String, BigDecimal> {
   }
 
   /**
-   * Return the additional (undeclared) property.
+   * Return the additional (undeclared) properties.
+   * @return the additional (undeclared) properties
    */
   @JsonAnyGetter
   public Map<String, BigDecimal> getAdditionalProperties() {
@@ -103,6 +107,8 @@ public class AdditionalPropertiesNumber extends HashMap<String, BigDecimal> {
 
   /**
    * Return the additional (undeclared) property with the specified name.
+   * @param key the name of the property
+   * @return the additional (undeclared) property with the specified name
    */
   public BigDecimal getAdditionalProperty(String key) {
     if (this.additionalProperties == null) {

--- a/samples/client/petstore/java/native-async/src/main/java/org/openapitools/client/model/AdditionalPropertiesObject.java
+++ b/samples/client/petstore/java/native-async/src/main/java/org/openapitools/client/model/AdditionalPropertiesObject.java
@@ -82,6 +82,9 @@ public class AdditionalPropertiesObject extends HashMap<String, Map> {
   /**
    * Set the additional (undeclared) property with the specified name and value.
    * If the property does not already exist, create it otherwise replace it.
+   * @param key the name of the property
+   * @param value the value value of the property
+   * @return self reference
    */
   @JsonAnySetter
   public AdditionalPropertiesObject putAdditionalProperty(String key, Map value) {
@@ -93,7 +96,8 @@ public class AdditionalPropertiesObject extends HashMap<String, Map> {
   }
 
   /**
-   * Return the additional (undeclared) property.
+   * Return the additional (undeclared) properties.
+   * @return the additional (undeclared) properties
    */
   @JsonAnyGetter
   public Map<String, Map> getAdditionalProperties() {
@@ -102,6 +106,8 @@ public class AdditionalPropertiesObject extends HashMap<String, Map> {
 
   /**
    * Return the additional (undeclared) property with the specified name.
+   * @param key the name of the property
+   * @return the additional (undeclared) property with the specified name
    */
   public Map getAdditionalProperty(String key) {
     if (this.additionalProperties == null) {

--- a/samples/client/petstore/java/native-async/src/main/java/org/openapitools/client/model/AdditionalPropertiesString.java
+++ b/samples/client/petstore/java/native-async/src/main/java/org/openapitools/client/model/AdditionalPropertiesString.java
@@ -82,6 +82,9 @@ public class AdditionalPropertiesString extends HashMap<String, String> {
   /**
    * Set the additional (undeclared) property with the specified name and value.
    * If the property does not already exist, create it otherwise replace it.
+   * @param key the name of the property
+   * @param value the value value of the property
+   * @return self reference
    */
   @JsonAnySetter
   public AdditionalPropertiesString putAdditionalProperty(String key, String value) {
@@ -93,7 +96,8 @@ public class AdditionalPropertiesString extends HashMap<String, String> {
   }
 
   /**
-   * Return the additional (undeclared) property.
+   * Return the additional (undeclared) properties.
+   * @return the additional (undeclared) properties
    */
   @JsonAnyGetter
   public Map<String, String> getAdditionalProperties() {
@@ -102,6 +106,8 @@ public class AdditionalPropertiesString extends HashMap<String, String> {
 
   /**
    * Return the additional (undeclared) property with the specified name.
+   * @param key the name of the property
+   * @return the additional (undeclared) property with the specified name
    */
   public String getAdditionalProperty(String key) {
     if (this.additionalProperties == null) {

--- a/samples/client/petstore/java/native/src/main/java/org/openapitools/client/model/AdditionalPropertiesAnyType.java
+++ b/samples/client/petstore/java/native/src/main/java/org/openapitools/client/model/AdditionalPropertiesAnyType.java
@@ -82,6 +82,9 @@ public class AdditionalPropertiesAnyType extends HashMap<String, Object> {
   /**
    * Set the additional (undeclared) property with the specified name and value.
    * If the property does not already exist, create it otherwise replace it.
+   * @param key the name of the property
+   * @param value the value value of the property
+   * @return self reference
    */
   @JsonAnySetter
   public AdditionalPropertiesAnyType putAdditionalProperty(String key, Object value) {
@@ -93,7 +96,8 @@ public class AdditionalPropertiesAnyType extends HashMap<String, Object> {
   }
 
   /**
-   * Return the additional (undeclared) property.
+   * Return the additional (undeclared) properties.
+   * @return the additional (undeclared) properties
    */
   @JsonAnyGetter
   public Map<String, Object> getAdditionalProperties() {
@@ -102,6 +106,8 @@ public class AdditionalPropertiesAnyType extends HashMap<String, Object> {
 
   /**
    * Return the additional (undeclared) property with the specified name.
+   * @param key the name of the property
+   * @return the additional (undeclared) property with the specified name
    */
   public Object getAdditionalProperty(String key) {
     if (this.additionalProperties == null) {

--- a/samples/client/petstore/java/native/src/main/java/org/openapitools/client/model/AdditionalPropertiesArray.java
+++ b/samples/client/petstore/java/native/src/main/java/org/openapitools/client/model/AdditionalPropertiesArray.java
@@ -83,6 +83,9 @@ public class AdditionalPropertiesArray extends HashMap<String, List> {
   /**
    * Set the additional (undeclared) property with the specified name and value.
    * If the property does not already exist, create it otherwise replace it.
+   * @param key the name of the property
+   * @param value the value value of the property
+   * @return self reference
    */
   @JsonAnySetter
   public AdditionalPropertiesArray putAdditionalProperty(String key, List value) {
@@ -94,7 +97,8 @@ public class AdditionalPropertiesArray extends HashMap<String, List> {
   }
 
   /**
-   * Return the additional (undeclared) property.
+   * Return the additional (undeclared) properties.
+   * @return the additional (undeclared) properties
    */
   @JsonAnyGetter
   public Map<String, List> getAdditionalProperties() {
@@ -103,6 +107,8 @@ public class AdditionalPropertiesArray extends HashMap<String, List> {
 
   /**
    * Return the additional (undeclared) property with the specified name.
+   * @param key the name of the property
+   * @return the additional (undeclared) property with the specified name
    */
   public List getAdditionalProperty(String key) {
     if (this.additionalProperties == null) {

--- a/samples/client/petstore/java/native/src/main/java/org/openapitools/client/model/AdditionalPropertiesBoolean.java
+++ b/samples/client/petstore/java/native/src/main/java/org/openapitools/client/model/AdditionalPropertiesBoolean.java
@@ -82,6 +82,9 @@ public class AdditionalPropertiesBoolean extends HashMap<String, Boolean> {
   /**
    * Set the additional (undeclared) property with the specified name and value.
    * If the property does not already exist, create it otherwise replace it.
+   * @param key the name of the property
+   * @param value the value value of the property
+   * @return self reference
    */
   @JsonAnySetter
   public AdditionalPropertiesBoolean putAdditionalProperty(String key, Boolean value) {
@@ -93,7 +96,8 @@ public class AdditionalPropertiesBoolean extends HashMap<String, Boolean> {
   }
 
   /**
-   * Return the additional (undeclared) property.
+   * Return the additional (undeclared) properties.
+   * @return the additional (undeclared) properties
    */
   @JsonAnyGetter
   public Map<String, Boolean> getAdditionalProperties() {
@@ -102,6 +106,8 @@ public class AdditionalPropertiesBoolean extends HashMap<String, Boolean> {
 
   /**
    * Return the additional (undeclared) property with the specified name.
+   * @param key the name of the property
+   * @return the additional (undeclared) property with the specified name
    */
   public Boolean getAdditionalProperty(String key) {
     if (this.additionalProperties == null) {

--- a/samples/client/petstore/java/native/src/main/java/org/openapitools/client/model/AdditionalPropertiesInteger.java
+++ b/samples/client/petstore/java/native/src/main/java/org/openapitools/client/model/AdditionalPropertiesInteger.java
@@ -82,6 +82,9 @@ public class AdditionalPropertiesInteger extends HashMap<String, Integer> {
   /**
    * Set the additional (undeclared) property with the specified name and value.
    * If the property does not already exist, create it otherwise replace it.
+   * @param key the name of the property
+   * @param value the value value of the property
+   * @return self reference
    */
   @JsonAnySetter
   public AdditionalPropertiesInteger putAdditionalProperty(String key, Integer value) {
@@ -93,7 +96,8 @@ public class AdditionalPropertiesInteger extends HashMap<String, Integer> {
   }
 
   /**
-   * Return the additional (undeclared) property.
+   * Return the additional (undeclared) properties.
+   * @return the additional (undeclared) properties
    */
   @JsonAnyGetter
   public Map<String, Integer> getAdditionalProperties() {
@@ -102,6 +106,8 @@ public class AdditionalPropertiesInteger extends HashMap<String, Integer> {
 
   /**
    * Return the additional (undeclared) property with the specified name.
+   * @param key the name of the property
+   * @return the additional (undeclared) property with the specified name
    */
   public Integer getAdditionalProperty(String key) {
     if (this.additionalProperties == null) {

--- a/samples/client/petstore/java/native/src/main/java/org/openapitools/client/model/AdditionalPropertiesNumber.java
+++ b/samples/client/petstore/java/native/src/main/java/org/openapitools/client/model/AdditionalPropertiesNumber.java
@@ -83,6 +83,9 @@ public class AdditionalPropertiesNumber extends HashMap<String, BigDecimal> {
   /**
    * Set the additional (undeclared) property with the specified name and value.
    * If the property does not already exist, create it otherwise replace it.
+   * @param key the name of the property
+   * @param value the value value of the property
+   * @return self reference
    */
   @JsonAnySetter
   public AdditionalPropertiesNumber putAdditionalProperty(String key, BigDecimal value) {
@@ -94,7 +97,8 @@ public class AdditionalPropertiesNumber extends HashMap<String, BigDecimal> {
   }
 
   /**
-   * Return the additional (undeclared) property.
+   * Return the additional (undeclared) properties.
+   * @return the additional (undeclared) properties
    */
   @JsonAnyGetter
   public Map<String, BigDecimal> getAdditionalProperties() {
@@ -103,6 +107,8 @@ public class AdditionalPropertiesNumber extends HashMap<String, BigDecimal> {
 
   /**
    * Return the additional (undeclared) property with the specified name.
+   * @param key the name of the property
+   * @return the additional (undeclared) property with the specified name
    */
   public BigDecimal getAdditionalProperty(String key) {
     if (this.additionalProperties == null) {

--- a/samples/client/petstore/java/native/src/main/java/org/openapitools/client/model/AdditionalPropertiesObject.java
+++ b/samples/client/petstore/java/native/src/main/java/org/openapitools/client/model/AdditionalPropertiesObject.java
@@ -82,6 +82,9 @@ public class AdditionalPropertiesObject extends HashMap<String, Map> {
   /**
    * Set the additional (undeclared) property with the specified name and value.
    * If the property does not already exist, create it otherwise replace it.
+   * @param key the name of the property
+   * @param value the value value of the property
+   * @return self reference
    */
   @JsonAnySetter
   public AdditionalPropertiesObject putAdditionalProperty(String key, Map value) {
@@ -93,7 +96,8 @@ public class AdditionalPropertiesObject extends HashMap<String, Map> {
   }
 
   /**
-   * Return the additional (undeclared) property.
+   * Return the additional (undeclared) properties.
+   * @return the additional (undeclared) properties
    */
   @JsonAnyGetter
   public Map<String, Map> getAdditionalProperties() {
@@ -102,6 +106,8 @@ public class AdditionalPropertiesObject extends HashMap<String, Map> {
 
   /**
    * Return the additional (undeclared) property with the specified name.
+   * @param key the name of the property
+   * @return the additional (undeclared) property with the specified name
    */
   public Map getAdditionalProperty(String key) {
     if (this.additionalProperties == null) {

--- a/samples/client/petstore/java/native/src/main/java/org/openapitools/client/model/AdditionalPropertiesString.java
+++ b/samples/client/petstore/java/native/src/main/java/org/openapitools/client/model/AdditionalPropertiesString.java
@@ -82,6 +82,9 @@ public class AdditionalPropertiesString extends HashMap<String, String> {
   /**
    * Set the additional (undeclared) property with the specified name and value.
    * If the property does not already exist, create it otherwise replace it.
+   * @param key the name of the property
+   * @param value the value value of the property
+   * @return self reference
    */
   @JsonAnySetter
   public AdditionalPropertiesString putAdditionalProperty(String key, String value) {
@@ -93,7 +96,8 @@ public class AdditionalPropertiesString extends HashMap<String, String> {
   }
 
   /**
-   * Return the additional (undeclared) property.
+   * Return the additional (undeclared) properties.
+   * @return the additional (undeclared) properties
    */
   @JsonAnyGetter
   public Map<String, String> getAdditionalProperties() {
@@ -102,6 +106,8 @@ public class AdditionalPropertiesString extends HashMap<String, String> {
 
   /**
    * Return the additional (undeclared) property with the specified name.
+   * @param key the name of the property
+   * @return the additional (undeclared) property with the specified name
    */
   public String getAdditionalProperty(String key) {
     if (this.additionalProperties == null) {

